stl/src/xstrxfrm.cpp

```diff
@@ -53,37 +53,38 @@ _EXTERN_C_UNLESS_PURE
 //     Non-standard: if OM/API error, return INT_MAX.
 _CRTIMP2_PURE size_t __CLRCALL_PURE_OR_CDECL _Strxfrm(
     char* _string1, char* _end1, const char* _string2, const char* _end2, const _Collvec* ploc) {

```

Comment:
While I recognize that I'm asking to grow the scope of this PR, I find the use of `_underscore_lowercase` names here disturbing. I always have to look up the conditions under which they're reserved - this case was no exception - and I think they contribute only confusion in this file.

Reply:
Cleansed all of src.